examples/exclude-words/create_wordlist.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import glob
+import json
+from collections import Counter
+from pathlib import Path
+from pdfminer.high_level import extract_text
+
+
+def main(args):
+    file_path = args.file_path
+    minimum_frequency = args.minimum_frequency
+    minimum_length = args.minimum_length
+    allow_numeric = args.allow_numeric
+    words_file = args.words_file
+
+    word_dict = Counter()
+    for file in file_path.iterdir():
+        if file.suffix != '.pdf':
+            continue
+        text = extract_text(file)
+        sentence_list = text.split('\n')
+        sentence_list = [sentence for sentence in sentence_list if sentence]
+
+        for sentence in sentence_list:
+            for word in sentence.split(' '):
+                word = word.lower()
+                word.replace(':', '')
```

Comment:
Why do we remove `:`? I'm not sure if this is universal and should always be removed. Can you make an optional argument to the parser for this? e.g. `--remove-characters abc:` (removes `a` `b` `c` `:`)

Reply:
When I tested, some documents had `iva:` and some documents had `iva` (without the `:`), and I wanted them both to count as `iva`, but making it an optional argument is a good idea!